test/package.js

```diff
@@ -21,3 +21,21 @@ test('Every rule is defined in index file', async t => {
 	t.is(Object.keys(index.configs.recommended.rules).length, ruleFiles.length,
 		'There are more exported rules in the recommended config than rule files.');
 });
+
+test('Every fixable rule got valid meta.type', async t => {
+	const validTypes = ['problem', 'suggestion', 'layout'];
+
+	const files = await pify(fs.readdir)('rules');
+	const ruleFiles = files.filter(file => path.extname(file) === '.js');
+
+	for (const file of ruleFiles) {
+		const name = path.basename(file, '.js');
+		const rule = index.rules[name];
+
+		t.true(rule.meta !== null, `${name} got no meta`);
```

Comment:
I don't think it's correct to check `null`, unless I'm mistaken, it would be `undefined` when not specified.

Reply:
Generally speaking, meta in rule seems to be mandatory, at least from the docs, so even a check for `undefined` shouldn't be needed as it will fail later anyhow. However, this guards us from reading property of `null` on the next line and outputs proper error instead of gibberish.

Reply:
Though I can remove it of course, it's that accessing by possibly null references makes me itchy.